userguide/dailyuse/waydroid.rst

```diff
@@ -0,0 +1,83 @@
+Android apps
+========================
+
+`Waydroid <https://waydro.id>`_ is a minimal Android container and compatibility layer to run Android apps on GNU/Linux operating systems such as Ubuntu Touch.
+
+.. note::
+    "Computer" refers to another device you connect your Ubuntu Touch device to (via USB here).
+    Your USB-attached computer must have ``adb`` and ``fastboot`` installed.
+
+.. note::
+    You will need to execute commands on your Ubuntu Touch device (and/or attached computer) to initialize Waydroid and install APKs.
+    You can do that by using the terminal application, but it is easier to use :doc:``adb shell </userguide/advanceduse/adb>`` or :doc:`set up SSH </userguide/advanceduse/ssh>` to access your Ubuntu Touch device from your computer.
```

Comment:
cool. I see just one more thing. the adb link doesn't work https://ci.ubports.com/job/docs.ubports.com/job/PR-561/11/artifact/_build/html/userguide/dailyuse/waydroid.html I think it misses a backtick

Reply:
darn it, i'll take a look

Reply:
it appears i had too many backticks